roles/tarsnap/files/bin/backup-pre/mysql.sh

```diff
@@ -4,4 +4,4 @@ TMPDIR=/var/lib/mysql/backup
 
 rm -rf "$TMPDIR"
 mkdir -p "$TMPDIR"
-mysql -e 'show databases' | while read DBNAME; do mysqldump -uroot --complete-insert --events --single-transaction "$DBNAME" > "$TMPDIR/$DBNAME".sql; done
+mysql -s -e 'show databases' | tail -n +1 | grep -v performance_schema | grep -v information_schema | while read DBNAME; do mysqldump -uroot --complete-insert --events --single-transaction "$DBNAME" > "$TMPDIR/$DBNAME".sql; done
```

Comment:
This failed due to: 

* MySQL displaying ASCII table (`-s`) 
* Trying to backup `performance_schema` and `information_schema`